brightlayer-ui/react-cli-templates
# Update root dev dependencies and silence eslint errors

#### Changes proposed in this Pull Request:
- Update root dev dependencies
- Silence the `@typescript-eslint/naming-convention` errors


#### To Test:
1. Build a new project for each ts template using one of the following commands:
- npx -p @brightlayer-ui/cli blui new react --template file:\<path-to-local-dev-folder\>/react-cli-templates/blank-typescript
- npx -p @brightlayer-ui/cli blui new react --template file:\<path-to-local-dev-folder\>/react-cli-templates/routing-typescript
- npx -p @brightlayer-ui/cli blui new react --template file:\<path-to-local-dev-folder\>/react-cli-templates/authentication-typescript

2. Build the project and ensure there are no linting errors
